src/main/java/thePackmaster/cards/coresetpack/BoosterTutor.java

```diff
@@ -31,59 +31,49 @@ public void update() {
                 isDone = true;
 
                 ArrayList<String> bannedPacks = new ArrayList<>();
-                ArrayList<AbstractCard> targets = new ArrayList<>(AbstractDungeon.player.drawPile.group);
+                ArrayList<AbstractCard> targets = new ArrayList<>();
+
+                for (AbstractCard c : AbstractDungeon.player.drawPile.group
+                ) {
+                    if ((c instanceof AbstractPackmasterCard) && c.rarity != CardRarity.BASIC) {
```

Comment:
Is checking `c instanceOf PackMasterCard` correct here? Some packs also contain base game cards

Reply:
Hmm... good point.  How do we assign a card's parent pack currently if its not a Packmaster Card?